portfolio/src/main/webapp/script.js

```diff
@@ -32,7 +32,7 @@ function addRandomQuote() {
  * Add all comments to the page
  */
 function addComments() {
-    var response_json = fetch('/data').then(response => response.json());
+    var response_json = fetch('/data?num_comments=' + document.getElementById("num_comments_field").value).then(response => response.json());
```

Comment:
What is the `'` after `num_comments=` used for?

Reply:
i had it there to close the string before appending the value after